archunit/src/main/java/com/tngtech/archunit/library/dependencies/Slice.java

```diff
@@ -44,6 +44,9 @@
  * com.mycompany.myapp.user
  * com.mycompany.myapp.authorization
  * </code></pre>
+ * Please, note that a {@link Slice} does not necesserlly represent set of subpackages inside the given root package,
+ * although it is often the case. The example above is just a common use case, but {@link Slice slice} can, technically,
+ * represent any set of {@link JavaClass JavaClasses}.
  * The top level packages under 'myapp' could be considered slices according to different domain aspects.<br>
  * Thus there could be a slice 'Order' housing all the classes from the {@code order} package, a slice 'Customer'
  * housing all the classes from the {@code customer} package and so on.
```

Comment:
As the rest of the JavaDoc still refers to the example in the `<code>` block:

How about moving your suggestion to a new paragraph at the end?
With tiny modifications, I'd suggest:
```suggestion
 * The top level packages under 'myapp' could be considered slices according to different domain aspects.<br>
 * Thus there could be a slice 'Order' housing all the classes from the {@code order} package, a slice 'Customer'
 * housing all the classes from the {@code customer} package and so on.
  * <p>
 * Note that a {@link Slice} does not necessarily represent a set of subpackages inside the given root package.
 * The example above is just a common use case, but {@link Slice slice} can technically represent any set
 * of {@link JavaClass JavaClasses}.
 * </p>
```

Reply:
Sounds good, I agree

Reply:
Thanks for the update! Could you still move the new paragraph to the end of the JavaDoc? (And should we close it with `</p>`?)

Reply:
Yeah, we should, sorry, missed that :)

Reply:
FYI: I've prepared the rebased commit with the tiny adaption as we agreed on branch `PR1366-mipo256` (I couldn't update yours). Maybe we'll find a way to include it in the upcoming release?

Reply:
Hey @hankem!
Yeah, I will provide the necessary changes very shortly

Reply:
Done.
Anything else @hankem ?

Reply:
Thanks for your effort on a Sunday! 
My last request (cf. https://github.com/TNG/ArchUnit/pull/1366#issuecomment-2598332165) is to rebase your branch on the current `main` branch on this https://github.com/TNG/ArchUnit/ repository to make the PR mergable.